Source/LinqToDB/Common/Configuration.cs

```diff
@@ -283,9 +283,14 @@ public static bool OptimizeJoins
 			}
 
 			/// <summary>
-			/// If set to true nullable fields would be checked for IS NULL in Equal/NotEqual comparisons.
+			/// If set to LikeCSharp nullable fields would be checked for IS NULL in Equal/NotEqual comparisons.
```

Comment:
```suggestion
			/// If set to <see cref="CompareNulls.LikeCSharp" /> nullable fields would be checked for IS NULL in Equal/NotEqual comparisons.
```
(and similar for other references to values from the enum)